bundles/org.openhab.voice.porcupineks/README.md

```diff
@@ -0,0 +1,25 @@
+# Porcupine Keyword Spotter
+
+This voice service allows you to use the PicoVoice product Porcupine as your keyword spotter in openHAB.
+
+## How to use it
+
+After installing you would be able to access the addon options through the openHAB configuration page under the 'Other Services' section. There you will need to provide your PicoVoice Api Key.
+
+After that you can select porcupine as your default Keyword Spotter in your 'Voice' settings.
+
+## Magic Word Configuration
+
+The magic word to spot is gathered from your 'Voice' configuration. The default english keyword models are loaded in the addon (also the english language model) so you can use those without adding anything else.
+
+Note that you can use the pico voice platform to generate your own keyword models, for use them you should place the generated file under '\<openHAB userdata\>/.porcupine' and configure your magic world to match the file name replacing spaces with '_' and adding the extension '.ppn'.
+
+As a summary service will only work if it's able find the correct ppn for your magic work configuration.
```

Comment:
As a summary**, the** service will only work if it's able **to** find the correct ppn for your magic **word** configuration.